vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -222,6 +222,7 @@ components:
       properties:
         size:
           type: integer
+          format: int64
```

Comment:
Shouldn't it be `uint64`?

Reply:
It seems to be an OpenAPI limitation:
https://swagger.io/docs/specification/data-models/data-types/#numbers